src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOneDay = 40;
+  const cost = days * costOneDay;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3 && days < 7) {
+    return cost - 20;
+  }
+
+  return cost;
```

Comment:
please use variables for constant numbers - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2